intercom_flutter/android/src/main/java/io/maido/intercom/PushInterceptService.java

```diff
@@ -1,25 +1,39 @@
 package io.maido.intercom;
-import com.google.firebase.messaging.*;
-import io.intercom.android.sdk.push.IntercomPushClient;
+
 import android.util.Log;
+
+import androidx.annotation.NonNull;
+
+import com.google.firebase.messaging.FirebaseMessagingService;
+import com.google.firebase.messaging.RemoteMessage;
+
+import org.jetbrains.annotations.NotNull;
+
 import java.util.Map;
 
+import io.intercom.android.sdk.push.IntercomPushClient;
+
 public class PushInterceptService extends FirebaseMessagingService {
-    private static final String TAG = "PushInterceptService";
-    private final IntercomPushClient intercomPushClient = new IntercomPushClient();
+  private static final String TAG = "PushInterceptService";
+  private final IntercomPushClient intercomPushClient = new IntercomPushClient();
 
-    @Override
-    public void onMessageReceived(RemoteMessage remoteMessage) {
+  @Override
+  public void onMessageReceived(@NotNull RemoteMessage remoteMessage) {
 
-        super.onMessageReceived(remoteMessage);
+    super.onMessageReceived(remoteMessage);
 
-        Map message = remoteMessage.getData();
+    Map<String, String> message = remoteMessage.getData();
 
-        if (intercomPushClient.isIntercomPush(message)) {
-            Log.d(TAG, "Intercom message received");
-            intercomPushClient.handlePush(getApplication(), message);
-        } else {
-            Log.d(TAG, "Push message received, not for Intercom");
-        }
+    if (intercomPushClient.isIntercomPush(message)) {
+      Log.d(TAG, "Intercom message received");
+      intercomPushClient.handlePush(getApplication(), message);
+    } else {
+      Log.d(TAG, "Push message received, not for Intercom");
     }
+  }
+
+  @Override
+  public void onNewToken(@NonNull @NotNull String s) {
```

Comment:
Is it required in Java to override a method to just call its super?

Reply:
yes, it is giving the warning.

Reply:
![Screen Shot 2021-08-02 at 3 04 51 PM](https://user-images.githubusercontent.com/14223997/127839584-022560b0-7cae-49c8-80ca-0ca92301da7f.png)